index.html

```diff
@@ -347,6 +347,311 @@ <h4>The encapsulateKey method</h4>
+              Let |normalizedDecapsulationAlgorithm| be the result of
+              <a data-cite="WebCryptoAPI#dfn-normalize-an-algorithm">normalizing an algorithm</a>, with
+              `alg` set to |decapsulationAlgorithm| and `op` set to
+              "`decapsulate`".
+            </p>
+          </li>
+          <li>
+            <p>
+              If an error occurred, return a Promise rejected with
+              |normalizedDecapsulationAlgorithm|.
+            </p>
+          </li>
+          <li>
+            <p>
+              Let |realm| be the [= relevant realm =] of [= this =].
+            </p>
+          </li>
+          <li>
+            <p>
+              Let |promise| be a new Promise.
+            </p>
+          </li>
+          <li>
+            <p>
+              Return |promise| and perform the remaining steps [= in parallel =].
+            </p>
+          </li>
+          <li>
+            <p>
+              If the following steps or referenced procedures say to
+              [= exception/throw =] an error,
+              [= queue a global task =] on the <a data-cite="WebCryptoAPI#dfn-crypto-task-source">crypto task source</a>,
+              given |realm|'s global object,
+              to reject |promise| with the returned error;
+              and then <a data-cite="WebCryptoAPI#terminate-the-algorithm">terminate the algorithm</a>.
+            </p>
+          </li>
+          <li>
+            <p>
+              If the {{Algorithm/name}} member of
+              |normalizedDecapsulationAlgorithm| is not equal to the
+              {{KeyAlgorithm/name}} attribute of the
+              <a data-cite="WebCryptoAPI#dfn-CryptoKey-slot-algorithm">`[[algorithm]]`</a> internal slot of
+              |decapsulationKey| then [= exception/throw =] an {{InvalidAccessError}}.
+            </p>
+          </li>
+          <li>
+            <p>
+              If the <a data-cite="WebCryptoAPI#dfn-CryptoKey-slot-usages">`[[usages]]`</a> internal slot of
+              |decapsulationKey| does not contain an entry that is "`decapsulateBits`",
+              then [= exception/throw =] an {{InvalidAccessError}}.
+            </p>
+          </li>
+          <li>
+            <p>
+              Let |decapsulatedBits| be the result of performing the decapsulate operation specified by
+              the <a data-cite="WebCryptoAPI#dfn-CryptoKey-slot-algorithm">`[[algorithm]]`</a> internal slot of |decapsulationKey| using |decapsulationKey| and |ciphertext|.
+            </p>
+          </li>
+          <li>
+            <p>
+              [= Queue a global task =] on the <a data-cite="WebCryptoAPI#dfn-crypto-task-source">crypto task source</a>,
+              given |realm|'s global object, to perform the remaining steps.
+            </p>
+          </li>
+          <li>
+            <p>
+              Let |result| be the result of converting |decapsulatedBits| to an ECMAScript Object in |realm|,
+              as defined by [[WebIDL]].
```

Comment:
I know that I wrote in the decapsulate operation that we'll return an ArrayBuffer, but I think it's wrong; we should probably return a byte sequence, like in the decrypt operation, and then here do:

```suggestion
                  Let |result| be the result of [= ArrayBuffer/create | creating =] an {{ArrayBuffer}}
                  in |realm|, containing |decapsulatedBits|.
```